README.md

```diff
@@ -4,7 +4,8 @@ Migrate devices using Carbonite Migrate.
 # User requirements
 ## The following software must be running/installed on the client computer:
 1. Device42 16.08.03+
-2.	Carbonite console 8.3.0.293+
+2. Carbonite console 8.3.0.293+
+3. **After installing the Carbonite Console, please copy the "DoubleTake.PowerShell.dll" from its install directory (usually C:\Program Files\Carbonite\Replication\Console directory) to the root folder of your scripts' location!**
```

Comment:
Need to renumber from here down.

Reply:
These number actually are just placeholders, below I just use all ones and when the file gets read, it automatically puts in the correct numbers.